src/TwinGet.Cli/TwinGet.Cli.csproj

```diff
@@ -6,9 +6,10 @@
     <AssemblyTitle>TwinGet Command Line</AssemblyTitle>
     <Description>TwinGet Command Line Interface</Description>
     <OutputType>Exe</OutputType>
-    <TargetFramework>net7.0</TargetFramework>
+    <TargetFramework>net7.0-windows</TargetFramework>

```

Comment:
Is the windows framework enforced by something?

Reply:
Beckhoff's Automation Interface can only be supported by Windows, so this whole project is limited to Windows platform.